bullet_train-super_scaffolding/Gemfile

```diff
@@ -11,5 +11,8 @@ gem "sprockets-rails"
 gem "bullet_train"
 gem "bullet_train-api"
 
+gem "masamune-ast" # TODO: Remove the bullet_train.gemspec dependency to super_scaffolding's gemspec.
```

Comment:
Just released [v1.2.1](https://github.com/gazayas/masamune-ast/pull/59) which includes https://github.com/gazayas/masamune-ast/pull/53

Reply:
Sweet, I just bumped the version in https://github.com/bullet-train-co/bullet_train-core/pull/598, then I can merge that and I can clean this up!